src/cli.rs

```diff
@@ -53,7 +53,7 @@ pub struct Cli {
     pub form: bool,
 
     /// Like --form, but force a multipart/form-data request even without files.
-    #[clap(short = 'm', long, overrides_with_all = &["json", "form"])]
+    #[clap(long, overrides_with_all = &["json", "form"])]
```

Comment:
This must have been an unintentional mistake since HTTPie never made use of the `-m` until now.